util/term/prompt.go

```diff
@@ -53,15 +53,15 @@ func PromptPassword(prompt string, allowEmpty bool) (string, error) {
 
 func Prompt(prompt string) (string, error) {
 	fmt.Print(prompt)
+
 	reader := bufio.NewReader(os.Stdin)
 	text, err := reader.ReadString('\n')
 
 	if err != nil {
 		return "", err
 	}
 
-	text = strings.Replace(text, "\n", "", -1)
+	text = strings.TrimSpace(text)
```

Comment:
What if a password has a trailing space?

Reply:
This isn't used for passwords. The password prompt method has had a trim for a long time. I think if they want to use a password with a space at the end or beginning, they need to use `-p`

Reply:
I wonder if spaces at the end of input are ever "desired" when not in the password. i checked the usages and it doesn't look it's used anywhere that would need a trailing space. since this function is public though - i think you should godoc this behavior - or gate it with a boolean that makes it obvious that the function will trim any input on your behalf so that the callee knows the input will be trimmed. my preference is the boolean.